Added review instructions to Contributing.md (#50)
project-codeflare/codeflare: CONTRIBUTING.md

## Reporting an Issue

To report an issue, or to suggest an idea for a change, open an [issue](../../issues/new). It is best to check our existing [issues](../../issues) first to see if a similar one has already been opened and discussed.

> ## Getting Feedback on your contribution

Reviewer: ```suggestion
## Getting Feedback on Your Contribution
```

When creating a PR for one of our projects it is important to get your work reviewed by other members of the team. Below are some steps to getting your work reviewed. <br>

> * Assign a Reviewer from the list of reviewers on the project

Reviewer: Maybe a first step could be to move the PR to the `Ready For Review` column on the sprint board so that the team can see that it is ready to be reviewed? 
Also you could add a bit more info to the other steps. Something like this
```suggestion
* Assign a Reviewer from the list of reviewers on the project
* Once the PR is reviewed the reviewer may request changes to be made, or if they are happy with the PR they will apply an `lgtm` label.
* When an `lgtm` label has been applied you can assign an Approver to your PR.
* The approver may request more changes, or if they are happy with the PR they will apply the `approve` label. After this time the PR will be merged.
```

Author: Good idea Fiona I'll add this in

## Reporting an Issue

To report an issue, or to suggest an idea for a change, open an [issue](../../issues/new). It is best to check our existing [issues](../../issues) first to see if a similar one has already been opened and discussed.
## Getting Feedback on Your Contribution

When creating a PR for one of our projects it is important to get your work reviewed by other members of the team. Below are some steps to getting your work reviewed. <br>
* Assign a Reviewer from the list of reviewers on the project.

> * Move the linked issue(s) from the `In Progress` column to `Ready For Review` column on the [Project Codeflare Sprint Board](https://github.com/orgs/project-codeflare/projects/8/views/3).

Reviewer 2: Nitpick 
This assumes the contributor is  using view 3, which you've linked. The default view is view 1 though, which doesn't have status columns. 

I'd suggest simply:
```suggestion
* Change the status of the issue from `In Progress` to `Ready For Review` on the [Project Codeflare Sprint Board](https://github.com/orgs/project-codeflare/projects/8/views/3).
```

It makes sense then for any view

Author: Done & changed the wording for the `In Review` and `Closed` statuses
